internal/updater/README.md

```diff
@@ -0,0 +1,60 @@
+* For the actual rollout we first need to generate a new key. That needs to be done by exactly one core maintainer with write access to the repo and the GHA secrets since only they can inject the new key, fingerprint and passphrase.
+* To generate the key run: `gpg --expert --full-generate-key` and select `RSA and RSA`, `3072` (bits) and a validity of `2y`. Use `Gopass Release Signing Key YYYY` as the name, `GitHub Actions only` as the comment and `release@gopass.pw` as the email.
+  * Note: If you're correctly following the 6 Months advance notice process, use the next year for the name.
+  * RSA isn't perfect but we used to have some compatability issues with non-RSA keys. Feel free to revisit this in the future. Keep the keep size to a reasonable value. Last time I checked 4096 did seem a bit excessive and with different algorithms these numbers will need to change as well. In doubt the `BSI TR-02102-1` should have a reasonable recommendation.
+  * Use a strong, random passphrase. Since you should never have to type it anywhere make it long, cryptic and stop worrying about it, e.g. `gopass pwgen 32`.
+* If you are me, you should probably also save a copy of both parts of new key into your gopass maintainer password store. For convenience add the Key ID / Fingerprint into the secret so you don't have to import the key just to get the Key ID.
+  * Hint: `gpg --output 0xKEYID.pub --armor --export 0xKEYID` and `gpg --output 0xKEYID.private --armor --export-secret-key 0xKEYID`
+* You should also sign the new key with the old key and possibly your personal key and push it to some keyservers.
+  * Use `gpg -u 0xOLDKEYID --edit-key 0xKEYID`, then sign.
+* Now export the public key and inject it into the pubkeys slice in `verify.go`. Add a comment with the year and the key id.
+* As usual send a PR and get this merged. Consider kicking off a new release, if that makes sense.
+* STOP HERE. For a seamless key rollover we need to wait until most users had a chance to update to a version that has both the old and the new keys. So if possible wait a few months at least. Keep the GH issue open and assigned to track that process.
+* After ~5 Months continue here.
+* Regenerate the test signature for verify_test.go
+  * Create a file that contains `gopass-sign-test\n` and run `gpg -u 0xKEYID --armor --output /tmp/testdata.sig --detatch-sign testdata`. Use the correct KEYID (the one of the NEW key).
+  * Hint: Make sure the input only contains one line break, not two.
+  * Paste the content of /tmp/testdata.sig into the `testSignature` in `verify_test.go`. Make sure all tests pass.
+* Navigate to https://github.com/gopasspw/gopass/settings/secrets/actions
+    * Paste the armored private part of the new key into the existing `GPG_PRIVATE_KEY` secret.
+    * Paste the corresponding passphrase into `PASSPHRASE`.
```

Comment:
BTW, I'm (no longer? or never was) not a maintainer, so I think you're the only one with access to these things for now.

Reply:
Maybe we should change that. But this discussion should not happen inside a PR. Maybe Slack or some other place.

Reply:
Yeah, wasn't intending to discuss that here, but wanted to raise the point that the bus factor/list of core maintainers is relatively low.